pkgs/http/lib/http.dart

```diff
@@ -25,7 +25,7 @@ export 'src/multipart_request.dart';
 export 'src/request.dart';
 export 'src/response.dart';
 export 'src/streamed_request.dart';
-export 'src/streamed_response.dart';
+export 'src/streamed_response.dart' hide StreamedResponseV2;
```

Comment:
[optional] Consider using a `show` instead. I don't think we've ever formalized it, but I think we've been moving in that direction for public libraries.

I'll probably at some point add explicit `show` clauses for all the exports here.

Reply:
Done